docs/sources/helm-charts/mimir-distributed/run-production-environment-with-helm/_index.md

```diff
@@ -342,3 +341,13 @@ rollout_operator:
 >   --set 'mimir-distributed.rollout_operator.podSecurityContext.runAsUser=null' \
 >   --set 'mimir-distributed.rollout_operator.podSecurityContext.runAsGroup=null'
 > ```
+
+{{% docs/reference %}}
+[Planning Grafana Mimir capacity]: "/ -> /docs/mimir/<MIMIR DOCS VERSION>/operators-guide/run-production-environment/planning-capacity"
+[Ingesters failure and data loss]: "/ -> /docs/mimir/<MIMIR DOCS VERSION>/references/architecture/components/ingester#ingesters-failure-and-data-loss"
+[Collecting metrics and logs from Grafana Mimir]: "/ -> /docs/mimir/<MIMIR DOCS VERSION>/operators-guide/monitor-grafana-mimir/collecting-metrics-and-logs"
+[Store-gateway: Blocks sharding and replication]: "/ -> /docs/mimir/<MIMIR DOCS VERSION>/references/architecture/components/store-gateway#blocks-sharding-and-replication"
+[replication across availability zones]: "/ -> /docs/mimir/<MIMIR DOCS VERSION>/configure/configure-zone-aware-replication"
+[Configure Grafana Mimir object storage backend]: "/ -> /docs/mimir/<MIMIR DOCS VERSION>/configure/configure-object-storage-backend"
+[Installing Grafana Mimir dashboards and alerts]: "/ -> /docs/mimir/<MIMIR DOCS VERSION>/operators-guide/monitor-grafana-mimir/installing-dashboards-and-alerts"
```

Comment:
```suggestion
[Planning Grafana Mimir capacity]: "/ -> /docs/mimir/<MIMIR DOCS VERSION>/manage/run-production-environment/planning-capacity"
[Ingesters failure and data loss]: "/ -> /docs/mimir/<MIMIR DOCS VERSION>/references/architecture/components/ingester#ingesters-failure-and-data-loss"
[Collecting metrics and logs from Grafana Mimir]: "/ -> /docs/mimir/<MIMIR DOCS VERSION>/manage/monitor-grafana-mimir/collecting-metrics-and-logs"
[Store-gateway: Blocks sharding and replication]: "/ -> /docs/mimir/<MIMIR DOCS VERSION>/references/architecture/components/store-gateway#blocks-sharding-and-replication"
[replication across availability zones]: "/ -> /docs/mimir/<MIMIR DOCS VERSION>/configure/configure-zone-aware-replication"
[Configure Grafana Mimir object storage backend]: "/ -> /docs/mimir/<MIMIR DOCS VERSION>/configure/configure-object-storage-backend"
[Installing Grafana Mimir dashboards and alerts]: "/ -> /docs/mimir/<MIMIR DOCS VERSION>/manage/monitor-grafana-mimir/installing-dashboards-and-alerts"
```

Reply:
This suggestion is not correct for the release of the Mimir documentation targeted by the Helm chart documentation. There is no "Manage" section in the `v2.9.x` documentation (https://grafana.com/docs/mimir/v2.9.x/).